src/common/utils/compare.ts

```diff
@@ -71,30 +62,6 @@ export function getAllCountiesOfState(stateCode: string): SummaryForCompare[] {
     .map(getLocationObj);
 }
 
-export function getAllMetroCounties(): SummaryForCompare[] {
```

Comment:
by code search and TSC, none of these were used, easy removal!
and since this was the only use of `getCountyMsaCode` now, we can get rid of that too.